flink-runtime-web/src/main/java/org/apache/flink/runtime/webmonitor/WebRuntimeMonitor.java

```diff
@@ -193,7 +193,12 @@ public WebRuntimeMonitor(
 		} else {
 			sslFactory = null;
 		}
-		metricFetcher = new MetricFetcher(retriever, queryServiceRetriever, scheduledExecutor, timeout);
+		metricFetcher = new MetricFetcher(
+			retriever,
+			queryServiceRetriever,
+			scheduledExecutor,
+			timeout,
+			WebOptions.METRIC_FETCHER_UPDATE_INTERVAL.defaultValue());
```

Comment:
Shouldn't it be the configured value instead of default value ?

Reply:
The `WebRuntimeMonitor` is legacy code which is no longer used.